input/haleu/README.rst

```diff
@@ -10,8 +10,8 @@ Inputs:
 This directory holds all of the input files for this project.
 
 The input files in the repository are the ``.xml`` input files for 
-|Cyclus|. The input files are titled by the type of advanced 
-reactors to be deployed and the demand growth of the scenario. 
+|Cyclus|. The input files are titled by scenario number, with 
```

Comment:
It's a good idea to come up with a naming convention, but I recommend making the names more informative than "scenarioX.xml". I will *have* to reference the table below in order to figure out which table is which. ALSO, what if you change the files and the results change? You'll have to update everything somehow which challenges reproducibility and code maintenance.

Just as a quick suggestion, what about something like:

``FC{OT or C}_AR0{0,1,2,3}_GPCT{0, 1, NA}_SC{#}_V{00X}.xml``

(fuel cycle) (advanced reactors) (growth percent) (scenario number) (version)

For scenario 2 this would look like 

``FCC_AR01_GPCT00_SC02_V001.xml``

It might be a bit more cumbersome, but once users become familiar with this naming convention, it will help them (and also possibly you) figure out what each input file is supposed to run. The table in this file is good and useful, but you need a stronger naming convention for your files. Also include a description of your naming convention -- and stick to it!

Reply:
This feels less intuitive to me, and your example for how it works is wrong, as Scenario 2 is a once-through fuel cycle. I feel like your convention would still require referencing back to a table of which  advanced reactor number corresponds with which advanced reactor. I'm also not sure why you would include a version number in the file name if it's all version controlled. 

I had previously used a convention of `reactor1_..._reactor_N_growth.xml`, which worked well for just the once-through scenarios. Adding in the closed fuel cycles, I felt that the file names would get too cumbersome and inconsistent (especially given that Scenarios 14 and 15 only differ by the material being reprocessed) and I felt like using the scenario numbers would be the easiest and cleanest. I understand that this may require users to reference the table in the README, but that's why I made sure to include the table as the input file names aren't descriptive without it.

Reply:
Apologies for the error in my example. The purpose of having a version number is in case you had multiple updated files. I don't think there's a "release" associated with this repository. It can be hard to reference a specific file version without a release tag.

Sure, it's cumbersome, but could be useful if there were many more input files than currently exist. You don't have to use it. However, I don't think users should be required to reference a readme in order to understand approximately what the file contains.